terraform/aks/workspace_variables/production_aks.tfvars.json

```diff
@@ -19,6 +19,8 @@
   "secondary_worker_replicas": 0,
   "clock_worker_replicas": 0,
   "postgres_flexible_server_sku": "GP_Standard_D2ds_v4",
+  "postgres_flexible_snapshot_server_sku": "B_Standard_B1ms",
+  "snapshot_databases_to_deploy": true,
```

Comment:
If the variable is changed to a boolean, could we make the name more explicit? e.g. `deploy_snapshot_databases`

Reply:
Yes much better name, made the change